container-images/tcib/base/openstackclient/openstackclient.yaml

```diff
@@ -16,11 +16,16 @@ tcib_packages:
   - python3-heatclient
   - python3-ironicclient
   - python3-manilaclient
-  - python3-observabilityclient
   - python3-octaviaclient
   - python3-aodhclient
   - bash-completion
   - iputils
+  #rhel: We can uncomment this when we have rhel specific pkgs
```

Comment:
what will be uncommented here?

do you mean more like "we can add rhel specific pkgs to this list" 

[EDIT]: ok i see what you mean now. we can uncomment "rhel:" specifically to add a new 'rhel' key and package list

Reply:
can I kindly ask for a consultation before you are actually removing packages that would affect other DFG?

Reply:
we shoudl not have rhel sepfcic packages in this file this is intended to work on both centos and rhel.

and we do not expect people to edit the file so i dont think this comment is relevent